### DISCUSSION TITLE
SpiralLinux on Vivobook - Ryzen 7, Geforce RTX, MT7921

### DISCUSSION THREAD

**Discussion by Topbanaana** [SpiralLinux/SpiralLinux-project#83] [Q&A]
Is there anybody with experience on Asus Vivobook here?

**Comment by geckolinux** [comment]
Hi there, thanks for the question. Do you own one yourself, or is this a pre-purchase question?

**Reply by Topbanaana** [reply]
I own one and use Budgie at present, to my satisfaction but with restrictions - some features are not working on Linux e.g. fingerprint reader on switch key, calculator on touchpad and some bluetooth devices are not recognized. I would make a try with SpiralLinux I think. In my working environment I use Debian on a ThinkPad, never had any issues and performance is excellent. 

A kernel version 5.10 is below requirement for this hardware with RTX and MT7921.

**Reply by geckolinux** [reply]
In that case it's definitely worth trying the live system. Another advantage of SpiralLinux over vanilla Debian in this case is that it includes the much newer kernel from Debian Backports (currently at 5.18) for newer hardware support.
